test/UnitTest/Microsoft.Test.AspNet.OData/PublicApi/PublicApiHelper.cs

```diff
@@ -402,11 +402,11 @@ private static bool AppendCustomAttributes(StringBuilder builder, object[] attri
                 }
                 foreach (object attribute in attributes)
                 {
-                    if (attribute is Runtime.InteropServices.MarshalAsAttribute)
+                    if (attribute is System.Runtime.InteropServices.MarshalAsAttribute)
```

Comment:
Can we use "Using System;"?

Reply:
For this issue, it's a bit more complicate since the new namespace starts with "Microsoft". The compiler searches up the namespace tree and never finds an assumes Runtime must be "Microsoft.Runtime". The file already has a "using System" but it doesn't help in this case.

That said, I can remove the partial namespace declarations and make them all usings.